pkg/hintrunner/zero/zerohint_keccak.go

```diff
@@ -485,3 +485,43 @@ func createBlockPermutationHinter(resolver hintReferenceResolver) (hinter.Hinter
 
 	return newBlockPermutationHint(keccakPtr), nil
 }
+
+// CompareBytesInWord hint compares a value to BYTES_IN_WORD constant, i.e., 8
+//
+// `newCompareBytesInWordHint` takes 1 operander as argument
+//   - `nBytes` is the value to be compared with BYTES_IN_WORD
+//
+// `newCompareBytesInWordHint` writes 1 or 0 to `ap` memory address depending on whether
+// `n_bytes` is lower than BYTES_IN_WORD or not
+func newCompareBytesInWordHint(nBytes hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "CompareBytesInWordHint",
+		Op: func(vm *VM.VirtualMachine, _ *hinter.HintRunnerContext) error {
+			//> ids.n_bytes < ids.BYTES_IN_WORD
+			nBytesVal, err := hinter.ResolveAsUint64(vm, nBytes)
```

Comment:
Should we not resolve this as a felt ? 

from cairo-lang : `n_bytes: felt`

Reply:
same, we can assume it should fit in uint64, will add a comment just in case.